### ISSUE TITLE
set hostname/machine name automativally

### ISSUE THREAD

**Issue by qgeissmann** [haneylab/pitally#2]
e.g. from the macaddress of eth0 port

**Comment by qgeissmann** [comment]
```python
a = netifaces.ifaddresses('eth0')[netifaces.AF_LINK][0]["addr"]
b = "".join(a.split(":")[3:6]) 
from subprocess import call
call("hostnamectl", "set-hostname",b)
```

**Comment by qgeissmann** [comment]
fixed by ecb2d66
